multiversx_sdk/network_providers/proxy_network_provider.py

```diff
@@ -181,7 +227,7 @@ def do_post_generic(self, resource_url: str, payload: Any) -> GenericResponse:
 
     def do_get(self, url: str) -> GenericResponse:
         try:
-            response = requests.get(url, auth=self.auth)
+            response = requests.get(url, auth=self.auth, **asdict(self.requests_config))
```

Comment:
As the other comment - tricky, opaque link (hard link) between the name of our field and the name of request's parameter.

Reply:
should be ok now